### PR TITLE
chore: replaced market name with tradable instrument name (1259)

### LINKED ISSUE #1259: Replace name in market queries with tradableInstrument
Replace `name` in market queries with:
```gql
tradableInstrument {
  instrument {
    code
    name
  }
}
```

Related to: https://github.com/vegaprotocol/vega/issues/6031

### PR DESCRIPTION
**Pull request by asiaznik**
# Related issues 🔗

Closes #1259
Related https://github.com/vegaprotocol/vega/issues/6031

# Description ℹ️

The [market framework spec](https://github.com/vegaprotocol/specs/blob/master/protocol/0001-MKTF-market_framework.md) does not mention a name at the top level definition of a market. There should only be a name and code at the instrument level.

### CODE REVIEW

**Review by bglownia — APPROVED** [review]

**Comment by edd** [comment]
I thought this might clash with https://github.com/vegaprotocol/frontend-monorepo/pull/1265, but it doesn't! Hooray.

File: apps/console-lite/src/app/components/deal-ticket/deal-ticket-steps.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ export const DealTicketSteps = ({
           ItemRenderer={MarketNameRenderer}
```
Comment (macqbat) [inline apps/console-lite/src/app/components/deal-ticket/deal-ticket-steps.tsx:179]: can't see related changes in mocks in unit tests and console-lite-e2e mock generators

File: apps/console-lite/src/app/components/simple-market-list/simple-market-renderer.tsx [hunk 2]
```diff
@@ -17,7 +17,9 @@ const MarketNameRenderer = ({ market, isMobile }: Props) => {
         )}
       />
       <div className="col-span-2 uppercase justify-start text-black dark:text-white text-ui-small md:text-market self-end">
-        {isMobile ? market.tradableInstrument.instrument.code : market.name}{' '}
+        {isMobile
```
Comment (macqbat) [inline apps/console-lite/src/app/components/simple-market-list/simple-market-renderer.tsx:20]: (non-blocking) wouldn't it be more elegant like `{market.tradableInstrument.instrument[isMobile ? 'code' : 'name']}`

File: apps/console-lite/src/app/components/deal-ticket/deal-ticket-steps.tsx (hunk 1 above)
Comment (asiaznik) [reply apps/console-lite/src/app/components/deal-ticket/deal-ticket-steps.tsx:179]: Done - perhaps it'd be good to consider coupling the mocks with generated types

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**